app/move/views/create/no-results.njk

```diff
@@ -0,0 +1,7 @@
+{% extends "form-wizard.njk" %}
+
+{% block contentMain %}
+Can't create new prisoners
```

Comment:
Just to be pedantic, we can put this in the locale files.

`{{ t("person::no_results_text") }}`

Reply:
Doh! These weren't meant to be committed! These are still WIP!

Reply:
Also, no pedantic at all. We should try to always use the translations for content.